aspnetcore/migration/50-to-60.md

```diff
@@ -114,14 +114,16 @@ There are a few changes to the other files generated for the Web App template:
 + public string? RequestId { get; set; }
 ```
 
-* `"Microsoft.Hosting.Lifetime": "Information"` was removed from both `appsettings.json` and `appsettings.Development.json`:
+* Log level defaults have changed in `appsettings.json` and `appsettings.Development.json`:
 
 ```diff
 - "Microsoft": "Warning",
 - "Microsoft.Hosting.Lifetime": "Information"
 + "Microsoft.AspNetCore": "Warning"
 ```
 
+In the preceding ASP.NET Core sample, `"Microsoft": "Warning"` has been changed to `"Microsoft.AspNetCore": "Warning"`. This change results in logging all informational messages from the `Microsoft` namespace ***except*** `Microsoft.AspNetCore`. For example, `Microsoft.EntityFrameworkCore` is now logged at the informational level.
```

Comment:
Its not a sample
Its template code

Reply:
Has the diff changed or is it just my mobile view?

Reply:
There is no difference in the diff of the template code.  However it shows red for the first two lines and green for the second as if the 2nd is new in GitHub.